src/main/java/cz/xtf/openshift/OpenshiftUtil.java

```diff
@@ -873,11 +873,13 @@ public void removeRoleFromServiceAccount(String namespace, String role, String s
 
 	private void removeRoleFromEntity(String namespace, String role, String entityKind, String entityName, String userName) {
 		withAdminUser(client -> {
-			if (client.inNamespace(namespace).roleBindings().withName(role).get() != null) {
-				client.inNamespace(namespace).roleBindings().withName(role).edit()
-								.removeFromSubjects(new ObjectReferenceBuilder().withKind(entityKind).withName(entityName).build())
-								.removeFromUserNames(userName)
-								.done();
+			RoleBinding roleBinding = client.inNamespace(namespace).roleBindings().withName(role).get();
```

Comment:
Have you considered something like ...ifPresent(rb -> rb.doMoreFunHere()) ? :)

Reply:
.withName(role) return type isn't Optional<RoleBinding> but Resource<RoleBinding, DoneableRolebinding>.

Reply:
Ehm, sorry my bad :)